packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -182,19 +182,6 @@ export const prodParasKusama: Omit<EndpointOption, 'teleport'>[] = [
       logo: nodesIpciSVG
     }
   },
-  {
```

Comment:
While still registered on-chain, see

![image](https://github.com/polkadot-js/apps/assets/1424473/b26b9e1c-3515-4ebc-b7bb-79215ad3fb2e)

The complete removal is a bit heavy-handed. All good removing the RPC endpoints, but the link from id -> name should stay in-tact.